src/components/Header/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,124 @@
+import { FC, useCallback, useEffect, useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import { Search } from "@mui/icons-material";
+import classNames from "classnames";
+
+import { GameType } from "@customTypes/Game";
+import { useOutsideClick } from "@hooks/useOutsideClick";
+
+import ImageWithLoader from "@components/ui/Image/ImageWithLoader";
+import { Loader } from "@components/ui/Loader/Loader";
+import { TLabel, TText } from "@components/ui/TranslationLabel/TLabel";
+
+import defaultGameThumbnailImage from "@assets/images/defaultGameThumbnail.png";
+
+import "./SearchBar.css";
+
+import { useGamesApi } from "@/api/services/gamesApi";
```

Comment:
```suggestion
import { useGamesApi } from "@api/services/gamesApi";
```
**Suggestion** : Remet à la bonne place